testutil/client.go

```diff
@@ -258,13 +258,16 @@ type LoginParams struct {
 // and returns the access JWT and refresh JWT extracted from
 // the HTTP response
 func HttpLogin(params *LoginParams) (*HttpToken, error) {
+	fmt.Println("HttpLogin: params data ", params)
 	loginPayload := api.LoginRequest{}
+	fmt.Println("HttpLogin: login payload at start", loginPayload)
 	if len(params.RefreshJwt) > 0 {
 		loginPayload.RefreshToken = params.RefreshJwt
 	} else {
 		loginPayload.Userid = params.UserID
 		loginPayload.Password = params.Passwd
 	}
+	fmt.Println("HttpLogin: login payload at end", loginPayload)
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein

note: need to rebase